### PR TITLE
Fix builds

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fix builds generating archives
| Type?         | bug fix
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | /
| How to test?  | All the builds are green and the module archive can be used on a shop

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -13,20 +13,42 @@ jobs:
...
+          yarn --cwd ~/PrestaShop/modules/blockwishlist run build
+          rm -rf node_modules
+
       - name: Install composer dependencies
         run: composer install --no-dev -o
+
       - name: Clean-up project
         uses: PrestaShopCorp/github-action-clean-before-deploy@v1.0
+        with: 
+          paths: node_modules
```
Comment (Quetzacoalt91) [inline .github/workflows/build-release.yml:44]: Folder node_modules is removed before zipping the module. Can you check I may do this @NeOMakinG ?

Comment (NeOMakinG) [reply .github/workflows/build-release.yml:44]: I think it's fine, webpack is getting every dependencies inside bundles, but we can also check if the module continue working without the folder after `npm run build`

File: .github/workflows/build-release.yml [hunk 2]
```diff
@@ -13,20 +13,42 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2.0.0
-      - name: Build JS dependencies
-        uses: PrestaShopCorp/github-action-build-js/12@v1.0
```
Comment (NeOMakinG) [inline .github/workflows/build-release.yml:17]: Hell, this is like I did, you'r forced to do the build without our github action ? :(

Comment (Quetzacoalt91) [reply .github/workflows/build-release.yml:17]: Because in a Docker context I loose the availability of the PrestaShop folder

**Pull request merged**